lib/player.js

```diff
@@ -1,4 +1,15 @@
 var Player = {
+	enable: function(socket) {
+		this.create_player(socket);
+
+		socket.on('set_initials', function(data){
+			if (data && data != socket.initials){
+				Player.update_initials(socket, data);
+				Player.reset_score(socket);
+			}
+			io.sockets.emit('user.renamed', socket.initials);
```

Comment:
Yes and no. It's good for testing but I think it's also nice if we want the server to be able to set data as well. It's a future proof thing.